samples/react-chat/src/App.tsx

```diff
@@ -13,18 +13,35 @@ export const ChatAddIcon = (): JSX.Element => {
   return <ChatAddIconBundle color={iconColor} />;
 };
 
-const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChatThread) => void }) => {
```

Comment:
Why remove the wrapper?

Reply:
I don't know if the wrapper provides any value, and I need easier access to `setShowNewChat(true)`